nilearn/surface/tests/_testing.py

```diff
@@ -4,6 +4,7 @@
 from scipy.spatial import Delaunay
 
 from nilearn.conftest import _rng
+from nilearn.surface import InMemoryMesh
 
 
 def generate_surf():
```

Comment:
Maybe for a post realease PR but I think that those functions could maybe be turned into fixtures, no?

Also it will be worth checking if they are not redundant with some of the fixtures we already have

See https://github.com/nilearn/nilearn/issues/4686

Reply:
Yes, we should do that. 

I actually like these better because they are more flexible -- we can create meshes with however many vertices, and the data can be more random.

The ones we currently use are more static.